src/main/java/assignment3/Application.java

```diff
@@ -0,0 +1,38 @@
+package assignment3;
+
+import assignment3.exceptions.CyclicDependencyException;
+import assignment3.exceptions.NodeAlreadyExistsException;
+import assignment3.exceptions.NodeNotFoundException;
+import assignment3.models.GraphModel;
+import assignment3.models.NodeModel;
+import assignment3.services.GraphService;
+
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+public class Application {
+
+  final static Logger logger = LoggerFactory.getLogger(GraphService.class);
+
+  /**
+   * Driver function
+   */
+  public static void init() {
```

Comment:
Please create a user manual, and take input from the user for each operation on family tree.